src/nidcpower/metadata/functions.py

```diff
@@ -1423,6 +1423,55 @@
         ],
         'returns': 'ViStatus'
     },
+    'GetChannelNameFromString': {
+        'documentation': {
+            'description': '\nReturns a comma-separated list of channel names from a string index list.'
```

Comment:
comma-separated list as in a string? is that right?
Why didn't we go with a list?

Reply:
We do return a `list` of `str`. Docstring needs to be updated.

Reply:
I matched what [we have for niDigital](https://github.com/ni/nimi-python/blob/master/src/nidigital/metadata/functions.py#L1963). Shouldn't they be the same?

Reply:
It's wrong there too. Created an issue for it - #1587. I'll fix it.

Reply:
Okay, I've changed it for nidcpower to "Returns a list of channel names for given channel indices" (in a separate PR)